src/syscalls.csv

```diff
@@ -0,0 +1,468 @@
+# name, index, x86_64, arm, aarch64, mips64n32, mips64, mips, parisc, ppc64, ppc, s390, s390x, x32, x86. riscv64
```

Comment:
```suggestion
# name, index, x86_64, arm, aarch64, mips64n32, mips64, mips, parisc, ppc64, ppc, s390, s390x, x32, x86, riscv64
```

Reply:
Also, thank you for adding RISC-V!  I wasn't going to ask you to do that, but that is nice to see :)

Reply:
Oh, one more thing - why do we need the second column, "index"?

Reply:
unfortunately gperf doesn't keep the same order when it generates its table.

I'll drop it from the csv and generate it